media-sound/qmmp/qmmp-1.1.10.ebuild

```diff
@@ -25,12 +25,13 @@ pulseaudio qsui qtmedia scrobbler sid sndfile soxr stereo tray udisks +vorbis wa
 
 REQUIRED_USE="gnome? ( dbus ) udisks? ( dbus )"
 
+# remove qt-5 subslot when minimum qt-5 version >= 5.9
 RDEPEND="
-	dev-qt/qtcore:5
-	dev-qt/qtgui:5
-	dev-qt/qtnetwork:5
-	dev-qt/qtwidgets:5
-	dev-qt/qtx11extras:5
+	dev-qt/qtcore:5=
```

Comment:
I think @Pesa 's idea was not to use = at all, but make this version depend on qt >= 5.9